### PR TITLE
Add logux_logger to configuration

### PR DESCRIPTION
**Pull request by wilddima**

### CODE REVIEW

File: lib/logux.rb [hunk 1]
```diff
@@ -51,11 +51,12 @@ def initialize(msg, meta: nil)
   configurable :logux_host, :verify_authorized,
                :password, :logger,
                :on_error, :auth_rule,
-               :render_backtrace_on_error
+               :render_backtrace_on_error, :logux_logger
```
Comment (dsalahutdinov) [inline lib/logux.rb]: let name it just `logger`? `Logux.logux_logger` sounds overloaded

Comment (wilddima) [reply lib/logux.rb]: But we already have logger: https://github.com/logux/logux_rails/blob/master/lib/logux.rb#L59-L62

Comment (gazay) [reply lib/logux.rb]: 👍

Comment (dsalahutdinov) [reply lib/logux.rb]: Adding one more logger looks confusedly. Why don't we just use the logger. If it is not set - instantiate our own?

Comment (wilddima) [reply lib/logux.rb]: @dsalahutdinov ok 👍

**Pull request merged**